src/gui/mod.rs

```diff
@@ -62,9 +62,10 @@ impl std::fmt::Display for AjourMode {
 }
 
 #[derive(Debug, Clone)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
@tarkah Clippy think we are using too big enum now. I ignored it. I think it's because we have `Addon` inside. You think it's gonna be a problem?

Reply:
I think it's fine until we finally get around to refactoring the entire GUI module where we can figure out how to better break everything up.